test/definitions/simple_workflow.json

```diff
@@ -6,6 +6,28 @@
   "version_minor": 0,
   "version_micro": 1,
   "tags": ["speech_to_text"],
+  "rights": [
+    {
+      "action": "view",
+      "groups": ["user"]
```

Comment:
define different groups per action to test filtering for each action.
Define also some common group for actions.

Create a more real example.